### PR TITLE
Add integer primitive operations

### PR DESCRIPTION
**Pull request by david-christiansen**
This makes integers more useful, now that we have them as of #89.

### CODE REVIEW

File: src/Expander.hs [hunk 1]
```diff
@@ -411,8 +411,34 @@ initializeKernel = do
             \(ValueString str2) ->
               ValueString (str1 <> str2)
         )
+      , ( "integer->string"
```
Comment (gelisam) [inline src/Expander.hs:414]: `int-ops` tests everything but `integer->string`, is that intentional?

File: stdlib/prelude.kl [hunk 2]
```diff
@@ -11,7 +11,7 @@
 (export -- primitive types
         Syntax
         ->
-        Integer
+        Integer + - * / abs negate > >= < <= = /=
```
Comment (gelisam) [inline stdlib/prelude.kl:14]: `integer->string` is again omitted. probably a mistake?

File: src/Expander.hs (hunk 1 above)
Comment (david-christiansen) [reply src/Expander.hs:414]: Nope, that was an oversight. Thanks!

File: stdlib/prelude.kl (hunk 2 above)
Comment (david-christiansen) [reply stdlib/prelude.kl]: Yep :)

**Comment by langston-barrett** [comment]
Looks great, thanks @david-christiansen!

**Pull request merged**